editor/components/SidebarLayout.tsx

```diff
@@ -68,7 +75,7 @@ export default function SidebarLayout({ children }: { children: ReactNode }) {
   }
 
   return (
-    <div style={{ height: "100vh", overflow: "hidden" }} className="d-flex">
+    <div style={{ overflow: "hidden" }} className="d-flex">
```

Comment:
idk what this height was doing before, bc i cant see any differences, but before removing this the scroll to top function would do nothing, something to do w [this](https://stackoverflow.com/questions/1174863/javascript-scrollto-method-does-nothing)?

Reply:
ahh and i see that on page refresh too the sidebar isnt taking up the full page until all the content loads. yeah i think a scroll to top was mentioned but i agree about nixing it, i forgot that was already built in to most browsers. but im glad i fiddled w it bc it made me abstract the rest of this feature for future fun shortcuts :))

Reply:
I think it was Xavier who said one can use the Home button to scroll to top, but he liked the idea of another way to scroll to top because the Home button was really far away.

Reply:
> but im glad i fiddled w it bc it made me abstract the rest of this feature for future fun shortcuts :))


same! 🚀